grunttasks/jscs.js

```diff
@@ -6,13 +6,22 @@ module.exports = function (grunt) {
   'use strict';
 
   grunt.config('jscs', {
-    src: [
-      '{,<%= yeoman.app %>/**/,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
-      '!<%= yeoman.app %>/bower_components/**',
-      '!<%= yeoman.app %>/scripts/vendor/**'
-    ],
-    options: {
-      config: '.jscsrc'
+    server: {
+      src: '{,tests/**/,grunttasks/**/,server/**/,scripts/**/}*.js',
+      options: {
+        config: '.jscsrc'
+      }
+    },
+    app: {
+      src: [
+        '{,<%= yeoman.app %>/**/}*.js',
```

Comment:
I *think* this can be simplified to `'<%= yeoman.app %>/**/*.js',` since the root folder is already being linted above in the `server` target glob.